build-native-image-windows.bat

```diff
@@ -3,15 +3,20 @@ set root=%~dp0
 call %root%gradlew.bat clean build
 echo making config directory
 mkdir %root%build\libs\META-INF\native-image
-set configPath=%root%build\libs\META-INF\native-image
+set configPath=%root%native-image-items\config
 echo running agentlib
-call java -agentlib:native-image-agent=config-output-dir=%configPath% -jar %root%build\libs\anime-cli-1.0-SNAPSHOT-all.jar
+@REM no args for base command, then add all subcommands also. potentially will need to run with any arguments also?
+call java -agentlib:native-image-agent=config-merge-dir=%configPath% -jar %root%build\libs\anime-cli-1.0-SNAPSHOT-all.jar
+call java -agentlib:native-image-agent=config-merge-dir=%configPath% -jar %root%build\libs\anime-cli-1.0-SNAPSHOT-all.jar login
+call java -agentlib:native-image-agent=config-merge-dir=%configPath% -jar %root%build\libs\anime-cli-1.0-SNAPSHOT-all.jar random
+call java -agentlib:native-image-agent=config-merge-dir=%configPath% -jar %root%build\libs\anime-cli-1.0-SNAPSHOT-all.jar random -u hone_the_rat -l completed
```

Comment:
see documentation on [agentlib](https://www.graalvm.org/22.0/reference-manual/native-image/Agent/)

> It can be necessary to run the target application more than once with different inputs to trigger separate execution paths for a better coverage of dynamic accesses. The agent supports this with the config-merge-dir option which adds the intercepted accesses to an existing set of configuration files

I read it and literally didn't even think about that until hours in 🤦